projects/lua/Dockerfile

```diff
@@ -0,0 +1,22 @@
+# Copyright 2020 Google Inc.
```

Comment:
nit: LLC not "Inc."

Reply:
Ah yeah - I started fixing this in my other commits too. Will ensure this one is fixed once the Lua people give confirm yes/no about integration!

Reply:
@jonathanmetzman documentation says "Google Inc.", not "Google LLC"  - https://google.github.io/oss-fuzz/getting-started/new-project-guide/#copyright-headers

Reply:
Changing to LLC based on this https://github.com/google/oss-fuzz/pull/4753

Reply:
Sorry documentation is wrong. I'll fix.